dev/networks: update public testnets (#6195)
ethereum/ethereum-org-website: src/content/developers/docs/networks/index.md

ETH on testnets has no real value; therefore, there are no markets for testnet ETH. Since you need ETH to actually interact with Ethereum, most people get testnet ETH from faucets. Most faucets are webapps where you can input an address which you request ETH to be sent to.

#### Sepolia {#sepolia}

A proof-of-work testnet; this means it's the best like-for-like representation of Ethereum.

- [Github](https://github.com/goerli/sepolia)
- [Otterscan](https://sepolia.otterscan.io/)

#### Görli {#goerli}

A proof-of-authority testnet that works across clients; an ideal testnet for application developers.

- [Github](https://github.com/goerli/testnet)
- [Etherscan](https://goerli.etherscan.io)

> Görli faucets:

Reviewer: ```suggestion
##### Görli faucets
```

- [Görli faucet](https://faucet.goerli.mudit.blog/)
- [Chainlink faucet](https://faucets.chain.link/)
- [Alchemy Goerli Faucet](https://goerlifaucet.com/)

#### Ropsten _(deprecated)_ {#ropsten}

> _Note, the Ropsten testnet is [deprecated](https://github.com/ethereum/pm/issues/460) and will no longer recieve protocol upgrades. Please consider migrating your applications to Sepolia or Goerli._

Reviewer: - Fix typo
- Make link more accessible
- Goerli -> Görli

```suggestion
_Note, [the Ropsten testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Görli._
```

> A very old proof-of-work testnet. It only serves historical relevance and is mainly used by core developers for testing protocol upgrades.

Reviewer: ```suggestion
A proof-of-work testnet. It only serves historical relevance and is mainly used by core developers for testing protocol upgrades.
```

> Ropsten faucets:

Reviewer: ```suggestion
##### Ropsten faucets
```

- [FaucETH](https://fauceth.komputing.org) (multi-Chain faucet without the need for social account)
- [Paradigm faucet](https://faucet.paradigm.xyz/)

#### Rinkeby _(deprecated)_ {#rinkeby}

> _Note, the Rinkey testnet is [deprecated](https://github.com/ethereum/pm/issues/460) and will no longer recieve protocol upgrades. Please consider migrating your applications to Sepolia or Goerli._

Reviewer: ```suggestion
_Note: [the Rinkeby testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Görli._
```

> A very old proof-of-authority testnet for those running old versions of the Geth client.

Reviewer: ```suggestion
A proof-of-authority testnet for those running old versions of the Geth client.
```

> Rinkeby faucets:

Reviewer: ```suggestion
##### Rinkeby faucets

```

- [FaucETH](https://fauceth.komputing.org) (multi-Chain faucet without the need for social account)
- [Alchemy faucet](https://RinkebyFaucet.com)
- [Chainlink faucet](https://faucets.chain.link/)
- [Paradigm faucet](https://faucet.paradigm.xyz/)
- [Rinkeby faucet](https://faucet.rinkeby.io/)

#### Kovan _(deprecated)_ {#kovan}

> _Note, the Kovan testnet is [deprecated](https://github.com/ethereum/pm/issues/460) and will no longer recieve protocol upgrades. Please consider migrating your applications to Sepolia or Goerli._

Reviewer: ```suggestion
_Note: [the Kovan testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Görli._
```

A very old proof-of-authority testnet for those still running OpenEthereum clients.

> Kovan faucets:

Reviewer: ```suggestion
##### Kovan faucets
```

- [FaucETH](https://fauceth.komputing.org) (multi-Chain faucet without the need for social account)
- [Chainlink faucet](https://faucets.chain.link/)
- [Paradigm faucet](https://faucet.paradigm.xyz/)

> ### Layer-2 Testnets {#layer-2-testnets}

Reviewer: ```suggestion
### Layer 2 testnets {#layer-2-testnets}
```

> [Layer 2 (L2)](/layer-2/) is a collective term to describe a specific set of Ethereum scaling solutions. A layer 2 is separate blockchain that extends Ethereum and inherits the security guarantees of Ethereum. Layer-2 testnets are usually tightly coupled to public Ethereum testnets.

Reviewer: ```suggestion
[Layer 2 (L2)](/layer-2/) is a collective term to describe a specific set of Ethereum scaling solutions. A layer 2 is a separate blockchain that extends Ethereum and inherits the security guarantees of Ethereum. Layer 2 testnets are usually tightly coupled to public Ethereum testnets.
```

##### Ropsten faucets

- [FaucETH](https://fauceth.komputing.org) (multi-Chain faucet without the need for social account)
- [Paradigm faucet](https://faucet.paradigm.xyz/)

#### Rinkeby _(deprecated)_ {#rinkeby}

_Note: [the Rinkeby testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Görli._

A proof-of-authority testnet for those running old versions of the Geth client.

##### Rinkeby faucets

- [FaucETH](https://fauceth.komputing.org) (multi-Chain faucet without the need for social account)
- [Alchemy faucet](https://RinkebyFaucet.com)
- [Chainlink faucet](https://faucets.chain.link/)
- [Paradigm faucet](https://faucet.paradigm.xyz/)
- [Rinkeby faucet](https://faucet.rinkeby.io/)

#### Kovan _(deprecated)_ {#kovan}

_Note: [the Kovan testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Görli._

A very old proof-of-authority testnet for those still running OpenEthereum clients.

> ##### Kovan faucets

Reviewer: ```suggestion
##### Kovan faucets

```
